setup.py

```diff
@@ -18,4 +22,8 @@
                    'Programming Language :: Python :: 3',
                    'Programming Language :: Python :: 3.5',
                    'Programming Language :: Python :: 3.6'],
-      install_requires=['scikit-learn', 'pandas'])
+      install_requires=['scikit-learn', 'pandas', 'forex_python'],

```

Comment:
Is it possible to indicate keras as an optional "requirement"?

Reply:
I could add something like:
```
extras_require={
     'keras':  ['keras']
}
```
Then, if you want to install that extra, you would have to write `pip install scikit-datasets[keras]`, or if another package depends on this functionality, it can add `scikit-datasets[keras]` to the list of dependencies.

I do not know if this is enough to install keras properly, though. Should we do that, maybe it is interesting to change the dependencies that are special for each repository,  such as `forex`, to extras, and provide documentation for them.

However, I am not sure of why do you want keras as an optional dependency. Have we a dataset that depends on keras?

Reply:
Great, let's do that, and let's also put forex in the same category.

MNIST, CIFAR and others are quite popular and are included in keras, that's why I added it as a dependency.